gubernator/github/handlers.py

```diff
@@ -29,13 +29,20 @@
 
 import classifier
 import models
+from .. import secrets
 
 
-try:
-    WEBHOOK_SECRET = open('webhook_secret').read().strip()
-except IOError:
-    logging.warning('unable to load webhook secret')
-    WEBHOOK_SECRET = 'default'
+WEBHOOK_SECRET = None
```

Comment:
Since this isn't a const I'd expect it to be `webhook_secret`

https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming